src/zdns/lookup_test.go

```diff
@@ -842,52 +843,6 @@ func TestNoResults(t *testing.T) {
 	verifyResult(t, *res, nil, nil)
 }
 
-// Test CName lookup returns ipv4 address
```

Comment:
Removed this because our CNAME following code is below the level of where the mockResults get returned in `doSingleDstServerLookup`.

Also, this is tested in `integration_tests.py` [here](https://github.com/zmap/zdns/pull/425/files#diff-eb8d4b2fb495641d278f4b64a596fdb9dd872986a38fb8d6478b775e6115ba4fR771)